### ISSUE TITLE
Unable to do anything (sync/send messages)

### ISSUE THREAD

**Issue by ositanachi** [nylas/N1#2211]
Sorry in advance if I'm doing any of filing an issue proccess wrong.
Version: 0.4.33-e8f137e

My system:
Manjaro Deepin
Manjaro Linux 16.03
Deepin Edition: 15.1.1 Desktop
System: 64 bit

I added my gmail account which seemed to have worked without a problem until I open N1 back. I am then greeted with "Nothing to display" (I added yahoo account that I do not use and saw the same thing). I tried sending a message with my gmail account and received an error message, "Sorry, this message could not be sent. Please try again, and make sure your message is addressed correctly and is not too large." 

I used Evolution with my gmail account and it worked fine (I do not have 2 step authentication enabled in case you are wondering). I'll update with logs as as it begins working again. For some reason the open detailed logs function is not working after I attempt to send an email.

**Comment by jstejada** [comment]
Hi @ositanachi, this might be the same issue as #2059. Can you try the fix that is suggested in that issue? If you continue having problems, can you send me your account email address info via slack? you can join here: http://slack-invite.nylas.com/

thanks!

**Comment by bengotow** [comment]
Handled in the slack channel - thanks!

**Comment by ositanachi** [comment]
Okie dokie, I finally got it working. @jstejada and @mbilker, thank you for your time and help. I could've sworn I installed gnome-keyring yesterday but I apparently did not. I'm pretty sure that was the issue since the only other thing I installed today was chromium. After installing gnome-keyring, I remove n1-git and its respective directory, installed n1 again, and viola it asked for a keyring after I selected the gmail account that I wanted to use.

I just sent an email to one of my other gmail accounts and it went through without a hitch. Thanks again!
